liqwid-script-export/example/Main.hs

```diff
@@ -0,0 +1,121 @@
+{-# OPTIONS_GHC -Wno-unticked-promoted-constructors #-}
+
+{- | Module     : Main
+     Maintainer : emi@haskell.fyi
+     Description: Example usage of 'liqwid-script-export'.
+
+Example usage of 'liqwid-script-export'.
+-}
+module Main (main) where
+
+import Data.Default (def)
+import Data.Map (fromList)
+import Data.Text (unpack)
+import Plutarch (compile)
+import Plutarch.Api.V2 (PValidator)
```

Comment:
Is this still using an old version of plutarch? Or did this module just not get updated?